pkg/gcloud/gcloud.go

```diff
@@ -61,14 +62,18 @@ func (g *gCloudSvc) Instances(serversMap map[string]server.Server) error {
 		log.Debug(len(instances))
 
 		for _, instance := range instances {
-			newServer := server.Server{}
-			newServer.Tags = make(map[string]string)
-			newServer.Name = instance.Name
-			newServer.Address = instance.NetworkInterfaces[0].AccessConfigs[0].NatIP
-			for index, key := range instance.Tags.Items {
-				newServer.Tags[strconv.FormatInt(int64(index), 10)] = key
+			if len(instance.NetworkInterfaces) > 0 {
+				newServer := server.Server{}
+				newServer.Tags = make(map[string]string)
+				newServer.Name = instance.Name
+				newServer.Address = instance.NetworkInterfaces[0].AccessConfigs[0].NatIP
```

Comment:
What happens if an Instance has a Network Interface that does not have an AccessConfig? We should iterate through all Network Interfaces and all AccessConfig Structs.